package.json

```diff
@@ -19,7 +19,9 @@
   ],
   "homepage": "https://github.com/mantoni/eslint_d.js",
   "scripts": {
-    "test": "eslint ."
+    "lint": "eslint bin lib",
```

Comment:
Probably should add the test directory to `.eslintignore` instead of this, better to be explicit about what not to lint.